sdk/servicebus/service-bus/test/streamingReceiverSessions.spec.ts

```diff
@@ -357,7 +357,7 @@ describe("Streaming with sessions", () => {
       const msgAbandonCheck = await checkWithTimeout(() => abandonFlag === 1);
       should.equal(msgAbandonCheck, true, "Abandoning the message results in a failure");
 
-      if (receiver.isReceivingMessages()) {
+      if ((receiver as any)._isReceivingMessages()) {
         await receiver.close();
       }
```

Comment:
This test can be updated with `subscriber.close()` instead of closing the receiver once @richardpark-msft's PRs are for `subscriber.close()` are checked in.